examples/python/IRD_stream.py

```diff
@@ -9,6 +9,7 @@
 # path
 basedir = pathlib.Path('/Users/yuikasagi/IRD/PhDwork/pyird/data/20210317/')
```

Comment:
Instead of this line, 
`basedir = pathlib.Path("~/pyird/data/20210317/").expanduser()`
makes no replacement of Path for the users/reviews/.

Reply:
That's a nice solution! Thank you, I'll change it.